### PR TITLE
Fixes #11902:  inventory from a node on Xen dom0 is missing UUID and is invalid

### PR DESCRIPTION
**Pull request by ncharles**
https://www.rudder-project.org/redmine/issues/11902

### CODE REVIEW

File: techniques/system/inventory/1.0/fusionAgent.st [hunk 1]
```diff
@@ -504,11 +502,11 @@ bundle agent addInformationsToInventory
     # define VMRUDDERUUID when we have xen
     xen.SuSE.(xen_domu_pv|xen_domu_hv)::
       "VMRUDDERUUID_cmd" string => "/bin/xenstore-read vm";
-    xen.((SuSE.xen_dom0)|centos|redhat)::
+    xen.(centos|redhat).!xen_dom0::
       "VMRUDDERUUID_cmd" string => "/usr/bin/xenstore-read vm";
-    xen.!SuSE.!centos.!redhat::
+    xen.!SuSE.!centos.!redhat.!xen_dom0::
```
Comment (peckpeck) [inline techniques/system/inventory/1.0/fusionAgent.st]: either .!SuSE must be remove or |SuSE muse be added in the previous case

Comment (peckpeck) [reply techniques/system/inventory/1.0/fusionAgent.st]: In fact all 3 cases are the same, they can be merged into a single one

Comment (peckpeck) [reply techniques/system/inventory/1.0/fusionAgent.st]: sorry not the same path

**Comment by ncharles** [comment]
OK, merging this PR

**Comment by ncharles** [comment]
OK, merging this PR

**Pull request merged**